conanfile.py

```diff
@@ -0,0 +1,44 @@
+from conans import ConanFile, CMake, tools
+
+class DbPostgreSQLAdapterConan(ConanFile):
+    name = "DbPostgreSQLAdapter"
+    description = "Implementation of C++ DB adapter based on PostgreSQL"
+    url = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    homepage = "https://github.com/systelab/cpp-postgresql-db-adapter"
+    author = "CSW <csw@werfen.com>"
+    topics = ("conan", "db", "sql", "postgreSQL", "adapter")
+    license = "MIT"
+    generators = "cmake_find_package"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "*", "!*.yml", "!*.md", "!.gitattributes", "!.gitignore", "!LICENSE"
+
+    def configure(self):
+        self.options["libpq"].shared = True
+        self.options["libpq"].with_openssl = True
+
+    def requirements(self):
+        self.requires("DbAdapterInterface/0.0.0@systelab/testing")
```

Comment:
Ups, good catch!

Reply:
I think conan requirements should not run under the assumption everyone has access to the your built packages.
All options necessary for building 'OpenSSL' and 'libpq' could be specified according to necessities of this project. conan install commands could be updated to build requirements in case of missing packages of required configuration. 

For 'DbAdapterInterface' I don't know what is the proper way to procede. Maybe a build and packaging of the library should be instantiated in 'DbPostgreSQLAdapterConan's recipe. (Not sure if this can be done comfortably). Maybe another option is to set 'DbAdapterInterface' as a submodule and run its conan install first.

Reply:
I understand. Right now we are on a tight schedule with this task, so I would prefer not to expend more time polishing it, but in a future, it could be interesting to package the sources so it could be recompiled.

There are a loot of suggestions on this PR indeed regarding CI good practices, and it is clear that something should be done any time soon

Reply:
Changed to 2.0.0 stable